run-tests.sh

```diff
@@ -2,39 +2,37 @@
 set -euo pipefail
 
 TESTING_BRANCH=${TESTING_BRANCH:-dev}
-KUBEADDONS_ENTERPRISE_PATH=${KUBEADDONS_ENTERPRISE_PATH:-kubeaddons-enterprise}
-KUBEADDONS_ENTERPRISE_TESTS_PATH="./tests"
-KUTTL_TEST_CONFIGURATION_PATH="kuttl-test.yaml"
-printf "testing kubeaddons-enterprise branch %s\n" "$TESTING_BRANCH"
+KUBEADDONS_REPO=${KUBEADDONS_REPO:-kubeaddons-enterprise}
+KUBEADDONS_TESTS_PATH="$(dirname $0)/tests"
+KUTTL_TEST_CONFIGURATION_PATH="${KUBEADDONS_TESTS_PATH}/${KUBEADDONS_REPO}/kuttl-test.yaml"
+printf "testing %s branch %s\n" "${KUBEADDONS_REPO}" "${TESTING_BRANCH}"
 
 if [ -d "./addons" ]
 then
-  printf "Running inside the kubeaddons-enterprise repository"
-  KUBEADDONS_ENTERPRISE_PATH="."
-  KUBEADDONS_ENTERPRISE_TESTS_PATH="kubeaddons-tests/tests/kubeaddons-enterprise"
-  KUTTL_TEST_CONFIGURATION_PATH="kubeaddons-tests/kuttl-test.yaml"
-elif [ -d "$KUBEADDONS_ENTERPRISE_PATH" ]
+  printf "Running inside the ${KUBEADDONS_REPO} repository"
+  KUBEADDONS_REPO="."
```

Comment:
```suggestion
  KUBEADDONS_TESTS_PATH="./kubeaddons-tests/tests/${KUBEADDONS_REPO}"
  KUBEADDONS_REPO="."
```
`KUBEADDONS_TESTS_PATH` should be set also to run tests from the addons repository, like `kubeaddons-enterprise` before we were setting it to `kubeaddons-tests/tests/kubeaddons-enterprise`

Reply:
We can't have tests in the repository or it breaks the old version of catalog that's in the wild.

Reply:
we don't have tests in the repository but it checks out this repository